model/chat.go

```diff
@@ -7,10 +7,11 @@ import (
 
 type Chat struct {
 	gorm.Model
-	UserID   string
-	UserName string
-	Message  string
-	StreamID uint
-	Admin    bool
-	SendTime time.Time
+
+	UserID   string    `gorm:"not null"`
+	UserName string    `gorm:"not null"`
+	Message  string    `gorm:"not null"`
+	StreamID uint      `gorm:"not null"`
+	Admin    bool      `gorm:"not null"`
+	SendTime time.Time `gorm:"not null"`
```

Comment:
While you're at it, can you remove sendTime and check that we just use gorms `createdAt`?

Reply:
Sure, no problem!